utils.R

```diff
@@ -127,7 +127,7 @@ download_cellset_file <- function(experiment_id) {
 #' @export
 download_processed_matrix <- function(experiment_id) {
   remote_path <-
-    paste0(paste("s3:/", "processed-matrix-production",
```

Comment:
Likewise, Can you refactor the account ID to a variable inside the file?

Reply:
done